com.playeveryware.eos/Runtime/Core/Config/ProductConfig.cs

```diff
@@ -63,7 +70,9 @@ public class ProductConfig : Config
         [ConfigField("Version",
             ConfigFieldType.Text,
             "Use this to indicate to the EOS SDK your game version.",
-            0)]
+            0, "https://dev.epicgames.com/docs/api-ref/structs/eos-initialize-options")]
+        [RegexValidation("^[A-Za-z0-9._ !?()+=:-]+$", "Product version must consist of only the following characters: A-Z, a-z, 0-9, dot, underscore, space, exclamation mark, question mark, sign, hyphen, parenthesis, plus, minus, or colon characters.")]
```

Comment:
It appears as though `=` also works.

Reply:
Well-spotted @WispyMouse! I've updated it to exclude that from the regex pattern.